### PR TITLE
fix(cli): Set `instance` when migrating Mintlify > Fern

### PR DESCRIPTION
**Pull request by mattblank11**
Related to [this task](https://linear.app/buildwithfern/issue/FER-3471/set-instance-when-migrating-mintlify-fern)

Here's what the `docs.yml` file looks like for Bland AI with the instances value set:
```
instances:
  - url: https://bland-ai.docs.buildwithfern.com
```

@coderabbitai summary

### CODE REVIEW

File: packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts [hunk 1]
```diff
@@ -95,6 +95,13 @@ export class MintlifyImporter extends DocsImporter<MintlifyImporter.Args> {
             }
         }
         this.context.logger.debug("Imported OpenAPI specs");
+
+        // We're currently setting one instance so that clients can generate docs out of the box,
+        // but they'll have the option to add additional instances and custom domains later.
+        builder.setInstance({
+            companyName: mint.name
+        });
+        this.context.logger.debug("Set instance");
```
Comment (RohinBhargava) [inline packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts:104]: do we want to output the name here, as well?

Comment (mattblank11) [reply packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts:104]: Good idea, updating now!

**Review by dsinghvi — APPROVED** [review]

File: packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts (hunk 1 above)
Comment (mattblank11) [reply packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts:104]: @RohinBhargava are you thinking that we should output the name from the mint file or the name we auto-generate?

Comment (RohinBhargava) [reply packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts:104]: good question! I think you have the most context, but I would assume `mint.name`, given the locality of the debug statement to the invocation

Comment (dsinghvi) [reply packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts:104]: @mattblank11 could see this being:  `Added instance ${url} to docs.yml`

Comment (mattblank11) [reply packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts]: Appreciate the input!

I agree with @dsinghvi on returning the url, if anything. IMO it's weird to return a string from the `setInstance` function but I don't love the alternative of creating a separate `getInstanceUrl` function. Feel free to push back on this!

**Pull request merged**